docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1842,6 +1848,31 @@ Comma-separated list of attributes to include in all new spans, such as `key1:va
 
 Can be set with the environment variable `OTEL_RESOURCE_ATTRIBUTES` (use `=` instead of `:` with the environment variable).
 
+### sampler_type
+
+Default value is `const`.
+
+Specifies the type of sampler: `const`, `probabilistic`, `ratelimiting`, or `remote`.
```

Comment:
```suggestion
This option specifies the type of sampler: `const`, `probabilistic`, `ratelimiting`, or `remote`.
```

Reply:
hmm... is it "option" or "setting"? I see both used in a few places in this doc, but most entries are just implicit (that's why I used the original form)

Reply:
Good point, it's probably "setting" but we can leave this as is so as not to introduce an inconsistency in the docs. I'm fine with "Specifies the type...".

Reply:
ok - TBH there's already inconsistency - maybe that's worth cleaning up _later_ 😉

I'll leave this one alone for now.

Reply:
👍